generator.py

```diff
@@ -20,7 +20,7 @@
             print("Parsing " + str(filename) + " language")
             lang = os.path.splitext(os.path.basename(filename))
             lang = lang[0]
-            json = load(f, object_hook=OrderedDict)
```

Comment:
I did not remove all the line, I just removed `object_hook=OrderedDict` because it is not necessary in Python >= 3.7.

More info [in the commit message](https://github.com/Mte90/espanso-typofixer/pull/5/commits/832c209dae6db219231319fd79c9e09e73cc0ada)

Reply:
We need it anyway as everyone can add words to the file and break the order (already happened in the past).